### PR TITLE
Feature/add proptypes

### PR DESCRIPTION
**Pull request by Fordo29**
## What Features are Being Added?
Added PropTypes to our functional components

## What is Being Refactored?
Added an h1 to our page and added a key to our favorites component and nav bar.

## Where should the reviewer start?
BookDetails.js, ErrorModal.js, BookContainer.js, BookCard.js

## How should this be manually tested?
Look for errors in console when navigating through app

## What are the relevant tickets?
Add proptypes ticket

## Any Other Notes/Questions/Problems?
Not at this time

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ class App extends Component {
           <Nav />
           <BookContainer allBooks={this.state.books}/>
         </Route>
-        <Route exact path='/favorites' render={() => {
-          return [<Nav location="favorites"/>, <Favorites />]
+        <Route exact path='/favorites' render={({ match }) => {
+          return [<Nav location="favorites" key={match + '-nav'}/>, <Favorites key={match + '-book-details'}/>]
```
Comment (cagallo) [inline src/components/App.js:42]: Brilliant Christine, thanks for fixing this bug!

File: src/components/BookCard.js [hunk 2]
```diff
@@ -18,3 +19,11 @@ const BookCard = ({title, author, bookImage, isbn, id }) => {
 }
 
 export default BookCard;
+
```
Comment (krdurrett) [inline src/components/BookCard.js:22]: These prop types are perfect

File: src/components/BookContainer.js [hunk 3]
```diff
@@ -23,3 +24,13 @@ const BookContainer = ({ allBooks }) => {
 }
 
 export default BookContainer;
+
+BookContainer.propTypes = {
```
Comment (cagallo) [inline src/components/BookContainer.js:28]: Great use of proptypes, super specific!

File: src/components/Nav.js [hunk 4]
```diff
@@ -6,9 +6,9 @@ import '../styles/Nav.css';
 const Nav = ({ location }) => {
   let navLink;
   if (location === 'favorites') {
-    navLink = <Link className="landing-page-link" to="/">Return Home »</Link>
+    navLink = <Link className="landing-page-link" to="/"><h1>Return Home »</h1></Link>
```
Comment (krdurrett) [inline src/components/Nav.js:9]: Nice add to fix some accessibility issues

**Review by krdurrett — APPROVED** [review]
Great work!

**Review by cagallo — APPROVED** [review]

**Pull request merged**